scripts/lcmcmc_fc.py

```diff
@@ -0,0 +1,343 @@
+import emcee
```

Comment:
add a docstring at the top of this file briefly explaining how the model works

Reply:
Done, give us a shout if it's not along the lines you were after